bootstrap/core/tools/argocd/argocd-clusterrolebindings.yaml

```diff
@@ -1,94 +1,52 @@
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  labels:
-    app.kubernetes.io/name: argocd-server
-    app.kubernetes.io/part-of: argocd
-    app.kubernetes.io/component: server
-  name: argocd-server
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: argocd-server
-subjects:
-- kind: ServiceAccount
-  name: argocd-server
-  namespace: argocd
----
-apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   labels:
-    app.kubernetes.io/name: argocd-server
+    app.kubernetes.io/managed-by: openshift-gitops
+    app.kubernetes.io/name: openshift-gitops
     app.kubernetes.io/part-of: argocd
-    app.kubernetes.io/component: server
-  name: argocd-server
+  name: argocd-application-controller
 rules:
 - apiGroups:
   - '*'
   resources:
   - '*'
   verbs:
-  - delete  # supports deletion a live object in UI
-  - get     # supports viewing live object manifest in UI
-  - patch   # supports `argocd app patch`
+  - '*'
```

Comment:
With this change, are any of the rules below required? It seems that this rule would make the things defined below redundant, but I could be wrong (it's been a while since I did RBAC).

Reply:
LGTM after internal discussion